### PR TITLE
KEYCLOAK-8810: Explain better how to preform a HttpServletRequest.logout()

### PR DESCRIPTION
**Pull request by rmartinc**
Explanation about how HttpServletRequest.logout() works when there is no refresh token to send to the server. The call is skipped and it can trigger some weird behavior when the logout app endpoint is not protected. As the page is unprotected, no security is performed to check for valid tokens and, therefore, the call maybe is skipped.

I preferred to just comment that using a protected URL to call the method is recommended.

### CODE REVIEW

File: securing_apps/topics/oidc/java/logout.adoc
Comment (matthewhelmke) [inline securing_apps/topics/oidc/java/logout.adoc:11]: I would like you to please edit this last bit to be like this:
For this reason, using a protected page to execute `HttpServletRequest.logout()` is recommended so that current tokens are always taken into account and an interaction with the {project_name} server is performed if needed.

**Review by matthewhelmke — CHANGES_REQUESTED** [review]
This is a nice addition, thank you! I have one simple edit request. Then, we need someone from engineering to review from their perspective. Once those are complete, we will be able to merge.

**Comment by rmartinc** [comment]
Done!

**Review by matthewhelmke — APPROVED** [review]
Excellent! Thank you. @stianst can you please assign someone to take a quick look. I estimate it will take 10 minutes or fewer.

**Review by mposolda — APPROVED** [review]
@matthewhelmke Approving as well. Thanks for the PR and Thanks @matthewhelmke  for the review.

**Pull request merged**
